lib/tizen_project.dart

```diff
@@ -69,8 +76,12 @@ class TizenProject extends FlutterProjectPlatform {
     if (!existsSync() || !isDotnet) {
       return;
     }
+    updateDotnetUserProjectFile();
+  }
 
-    final File userFile = editableDirectory.childFile('Runner.csproj.user');
+  void updateDotnetUserProjectFile() {
```

Comment:
Please consider making this method private.

```suggestion
  void _updateDotnetUserProjectFile() {
```